### PR TITLE
Created primenumbers_from_a_to_b_nestedforloop(Method_1).cpp

### LINKED ISSUE #479: Prime Number in range ( Algorithm )
In this i will like to find the prime numbers in a specific range from a to b and print the count and  numbers !! Will start working as soon as i get assigned !!

### PR DESCRIPTION
**Pull request by rohan-kulkarni-25**
Closed #479

### CODE REVIEW

File: Code/C++/primenumbers_from_a_to_b_nestedforloop(Method_1).cpp [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+*/
+
+/*
+
+Time Complexity :- O(N^2)
+
+*/
+
+/*
+Space Complexity :- O(2)
```
Comment (rudrakshi99) [inline Code/C++/primenumbers_from_a_to_b_nestedforloop(Method_1).cpp:114]: It should be O(1).

**Comment by rudrakshi99** [comment]
Delete the commit for Created primenumbers_till_n_nestedforloop.cpp as you have already committed the code for this in #545

**Comment by rohank2502** [comment]
> 
> 
> Delete the commit for Created primenumbers_till_n_nestedforloop.cpp as you have already committed the code for this in #545

Done !!

**Comment by tarun26091999** [comment]
@rudrakshi99 look into it.

**Comment by rish-singhal** [comment]
invalid PR

**Pull request closed without merging**